openmc/deplete/results_list.py

```diff
@@ -122,8 +123,8 @@ def get_reaction_rate(self, mat, nuc, rx):
 
         Parameters
         ----------
-        mat : str
-            Material name to evaluate
+        mat : openmc.Material
```

Comment:
Same idea here, need to have it accept either string or `Material` instance

Reply:
Yes good idea, I have done as requested